tests/test_mplcursors.py

```diff
@@ -304,6 +304,19 @@ def test_linecollection(ax):
     assert cursor.selections[0].target.index == approx((0, .5))
 
 
+def test_patchcollection(ax):
+    recs = [mpl.patches.Rectangle(xy, 0.1, 0.2)
+            for xy in zip((.1, .45, .7), (.1, .4, .7))]
```

Comment:
Regardless of what behavior we pick at the end, it may be nice to make the test use more "obvious" values (e.g. rectangles of size (1, 1) starting at (0, 0), (2, 2), (4, 4)) so that one can immediately see whether the mouse event synthesized below falls in one of the rectangles or on an edge or outside of all of them.